anchore_engine/services/catalog/catalog_impl.py

```diff
@@ -2517,4 +2518,33 @@ def add_event_json(event_json, dbsession, quiet=True):
             raise
 
 
+def load_policy_bundles(config, process_bundle, process_exception):
+    """
+    A convenience function to avoid code duplication between accounts.py and catalog/__init.py. This
+    function iterates through the one to many policy bundle filepaths in the config, opens them, and
+    converts them to json. Since the (currently two) calling methods do slightly different things with
+    those bundles (and handle exceptions in slightly ways) this function requires two callbacks it can
+    call to do that processing.
+
+    :param config The config:
+    :param process_bundle A callback with logic for what to do with each bundle:
```

Comment:
FWIW I'm generally avoid callbacks because I think it can make the code harder to read (although these are pretty lightweight). If in the future we can further refactor `accounts.py` and move more of that logic into the `catalog` we could get rid of these.